metadata_updater/check_duplicates.py

```diff
@@ -0,0 +1,67 @@
+import xml.etree.ElementTree as ET
+import os
+import csv
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+output_dir = 'C:/Users/ECheng/OneDrive - Land Information New Zealand/Desktop/Repo/Output' 
```

Comment:
I thought this was where an output was going to be stored, rather than the location of the input metadata.
It might be worth changing the variable name and adding a comment.